assets/dev/js/frontend/handlers/section.js

```diff
@@ -111,7 +111,7 @@ const BackgroundVideo = elementorModules.frontend.handlers.Base.extend( {
 						case YT.PlayerState.ENDED:
 							self.player.seekTo( elementSettings.background_video_start || 0 );
 							if ( elementSettings.background_play_once ) {
-								self.player.stopVideo();
+								self.player.destroy();
```

Comment:
Why?

Reply:
in order to remove the iframe completely